cmd/state-installer/installer.go

```diff
@@ -135,11 +130,13 @@ func install(installPath string, cfg *config.Instance, out output.Outputer) erro
 		return errs.Wrap(err, "Could not update PATH")
 	}
 
-	rcFile, err := shell.RcFile()
-	if err == nil {
-		out.Notice(fmt.Sprintf("Please either run 'source %s' or start a new login shell in order to start using the State Tool executable.", rcFile))
-	} else {
-		out.Notice("Please start a new login shell in order to start using the State Tool executable.")
+	if !funk.Contains(strings.Split(os.Getenv("PATH"), string(os.PathListSeparator)), installPath) {
+		rcFile, err := shell.RcFile()
+		if err == nil {
+			out.Notice(fmt.Sprintf("Please either run 'source %s' or start a new login shell in order to start using the State Tool executable.", rcFile))
```

Comment:
```suggestion
			out.Notice("Please start a new shell in order to start using the State Tool executable.")
```

Let's stick with telling them to restart their shell. This is assuming everyone has their rcfile set up in a way that you can re-source it without adverse effects.